build.gradle

```diff
@@ -48,7 +48,7 @@ publishing {
 repositories {
     mavenLocal()
     // Remove the commented code below once TransportService is published to maven
-    //maven { url "https://aws.oss.sonatype.org/content/repositories/snapshots" }
+//    maven { url "https://aws.oss.sonatype.org/content/repositories/snapshots" }
```

Comment:
At this point all changes to this file are whitespace. Would prefer if you can just revert this to the state before your PR.

Reply:
Done